src/webui/www/private/scripts/client.js

```diff
@@ -1563,6 +1563,7 @@ window.addEventListener("DOMContentLoaded", () => {
                 const id = "uploadPage";
                 new MochaUI.Window({
                     id: id,
+                    icon: "images/qbittorrent-tray.svg",
```

Comment:
I added it like this to all mocha windows but down the line common window properties / callbacks should probably be contained in some base object and then spread during window creation.